### ISSUE TITLE
Updated to v3.0.0-alpha.12.5 but still looks like v3.0.0-alpha.12.1.4

### ISSUE THREAD

**Issue by adamkhan** [strapi/strapi#1442]
**Informations**
- **Node.js version**: v10.1.0
- **npm version**: 6.1.0
- **Strapi version**: 3.0.0-alpha.12.15
- **Database**: MongoDB
- **Operating system**: MacOS

**What is the current behavior?**

After updating both local and global installs of Strapi on my dev machine to v3.0.0-alpha.12.5, I don't see any of the new things discussed at https://blog.strapi.io/release-v3-alpha-12-5-search-filters-bulk-actions/.

![screen shot 2018-06-21 at 16 01 23](https://user-images.githubusercontent.com/776967/41727588-9ac8e9dc-756c-11e8-86c2-c9f81180758c.png)

After running "strapi start" the Terminal does indeed output that we are running Version 3.0.0-alpha.12.5 (node v10.1.0), but when viewing a list of entries I still see "Content Manager' as the header instead of what appears in the demo video on the blog post (ie, the title of the particular individual content channel, there's no checkbox in the first column, no search box up top).

![screen shot 2018-06-21 at 16 02 55](https://user-images.githubusercontent.com/776967/41727618-a9b55462-756c-11e8-98f9-63b3185a85aa.png)

**Steps to reproduce the problem**

**What is the expected behavior?**

I expected to see the those new features discussed and illustrated at https://blog.strapi.io/release-v3-alpha-12-5-search-filters-bulk-actions/ after updating to v3.0.0-alpha.12.5.

**Suggested solutions**

Presumably even though the Terminal is telling me I'm running v3.0.0-alpha.12.5 I guess I'm stll running 12.1.4?

**Comment by krestaino** [comment]
Follow this.

https://github.com/strapi/strapi/wiki/migration-guide-alpha.12.4-to-alpha.12.5

You need to replace the old admin with the new one, as well as the plugins.

**Comment by adamkhan** [comment]
@krestaino Thanks for the quick response. Followed your link and successfully completed the update — thank you very much!

Might be good if the blog post linked to it as well, someone has already commented there about the issue.

**Comment by krestaino** [comment]
Agreed. Good news is that soon the updates should be much less of a manual process.

https://twitter.com/pierre_burgy/status/1009080766402322432
